### ISSUE TITLE
Author information is not shown

### ISSUE THREAD

**Issue by isuruf** [conda-forge/blog#22]
Also, there is a link at the end to another post without any explanation.

**Comment by isuruf** [comment]
cc @beckermr

**Comment by isuruf** [comment]
cc @beckermr

**Comment by beckermr** [comment]
Agreed about the links on the bottom. The author for each post is oddly in the sidebar.

<img width="1013" alt="Screen Shot 2020-06-04 at 12 05 26 AM" src="https://user-images.githubusercontent.com/5296416/83717030-34d37580-a5f7-11ea-973d-b76f0f092220.png">
